src/metal/window-proxy.ts

```diff
@@ -43,19 +47,35 @@ let W: WindowProxy = {
   getScrollLeft: nop,
   getHeight: nop,
   getWidth: nop,
+  rootsList: [],
   onWindowIsDirtyListeners: [],
   rAF: hasRAF ? window.requestAnimationFrame.bind(window) : (callback: Function) => { callback(); },
+  bindRootScrollEvent: (root: Element) => { root.addEventListener('scroll', () => W.callIsDirtyListeners(), false); },
```

Comment:
two questions:

1. should this be throttled
2. for nested elements, ones that are contained within occlusion culling how does this work?